db/migrate/20210609093404_drop_override_taxonomies_from_filter.rb

```diff
@@ -0,0 +1,8 @@
+class DropOverrideTaxonomiesFromFilter < ActiveRecord::Migration[6.0]
+  def up
+    perms = Permission.where(name: %w[view_filters create_filters edit_filters destroy_filters])
+    ids = Filter.joins(:filterings).where(filterings: { permission_id: perms }).pluck(:id).uniq
+    Filter.where(id: ids).update_all(override: false)
+    TaxableTaxonomy.where(taxable_type: 'Filter', taxable_id: ids).destroy_all
```

Comment:
Not crucial, but how about:
```suggestion
    filters = Filter.joins(:filterings).where(filterings: { permission_id: perms })
    filters.update_columns(override: false)
    TaxableTaxonomy.where(taxable: filters).destroy_all
```

Reply:
I do not trust Rails polymorphic association with subqueries, but might be just because of historically they didn't work well, they might do work great now :shrug: 
Lets try

Reply:
Works nice - at least in this case :)